File: 7KYU/Sum of numbers from 0 to N/solution.cs
using System.Linq;
using System;
public class SequenceSum
{
  public static string ShowSequence(int n)
  {    
    if(n < 0)
      return $"{n}<0";
    else if(n == 0)
      return "0=0";
    else
    {
      var range = Enumerable.Range(0, n+1);
      var total = range.Sum();
      return string.Join("+", range) + $" = {total}";
    }
  }
}

File: 7KYU/Sum of numbers from 0 to N/solution.js
var SequenceSum = (function() {
    function SequenceSum() {}
  
    SequenceSum.showSequence = function(count) {
      if(count < 0){
        return `${count}<0`;
      } else if (count ===  0) {
        return '0=0';
      } else {
        var res = "0";
        var total = 0;
        for(var i = 1; i <= count; i++){
          res += '+' + i;
          total+=i;
        }
        
        return res + ' = ' + total;
      }
    };
  
    return SequenceSum;
  
})();